components/table-list/src/table-list/table-list.types.ts

```diff
@@ -7,14 +7,24 @@ export type TableListProps = {
   className?: string;
 
   /**
-   * Alternating row colors variant.
+   * Table style variants.
+   * @default 'default'.
    */
-  isAlternating?: boolean;
-
+  variant?: 'default' | 'alternating' | 'borderless';
   /**
-   * Table title (caption).
+   * Table title (accessible caption).
    */
   caption?: string;
 
+  /**
+   * Accessible description for the table.
+   */
+  description?: string;
+
+  /**
+   * Table column widths.
+   */
+  columnWeights?: (number | string)[];
```

Comment:
did you mean 'ColumnWidths' for this?

Reply:
Hmm good point… It is basically the same thing as [grid-template-columns](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns), so maybe something like `columnSizing` is a better name 🤔